qa/vagrant/build.gradle

```diff
@@ -208,9 +208,19 @@ task checkVagrantVersion(type: Exec) {
   standardOutput = new ByteArrayOutputStream()
   doLast {
     String version = standardOutput.toString().trim()
-    if ((version ==~ /Vagrant 1\.[789]\..+/) == false) {
-      throw new InvalidUserDataException(
-        "Illegal version of vagrant [${version}]. Need [Vagrant 1.7+]")
+    if ((version ==~ /Vagrant 1\.(8\.[6-9]|9\.[0-9])+/) == false) {
+      throw new InvalidUserDataException("Illegal version of vagrant [${version}]. Need [Vagrant 1.8.6+]")
+    }
+  }
+}
+
+task checkVirtualBoxVersion(type: Exec) {
+  commandLine 'vboxmanage', '--version'
+  standardOutput = new ByteArrayOutputStream()
+  doLast {
+    String version = standardOutput.toString().trim()
+    if ((version ==~ /5\.[1-9]\..+/) == false) {
```

Comment:
I think we should do something slightly different here. This will break if a new major release (major version 6) is made, and will also break if there is a minor release in the major release 5 series with minor version 10 or greater (e.g., 5.10). I think that we should parse the version into major/minor components and require that major >= 5 and if major == 5 then minor > 0.